cookbooks/fastscape_eroding_box.prm

```diff
@@ -0,0 +1,172 @@
+# At the top, we define the number of space dimensions we would like to
+# work in:
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 2.5e5
+set Maximum time step                      = 50000 #5e6
```

Comment:
Please remove all commented values from the file (unless they serve a purpose, then add a sentence on what the commented value means).